src/views/Dashboard/components/TopCards.tsx

```diff
@@ -48,11 +53,15 @@ const TopCards: React.FC = () => {
     const factor = await getScalingFactor(yam);
     const projectedRebase = await getProjectedRebase(yam);
     const projectedMint = await getProjectedMint(yam);
-    setCurrentPrice(numeral(bnToDec(price)).format("0.0a"));
-    setScalingFactor(numeral(bnToDec(factor)).format("0.0a"));
+    const projectedRebasePercent = await getProjectedRebasePercent(yam);
+    const mintPercent = await getMintPercent(yam);
+    setCurrentPrice(numeral(bnToDec(price)).format("0.00a"));
+    setScalingFactor(numeral(bnToDec(factor)).format("0.00a"));
     setProjectedRebase((Math.sign(projectedRebase) === 1 ? "+" : "") + numeral(projectedRebase).format("0.0a"));
-    // setProjectedMint(numeral(projectedMint).format("0.0a"));
-  }, [setCurrentPrice, setScalingFactor, setProjectedRebase, , setProjectedMint, yam]);
+    //setProjectedMint(numeral((projectedMint)).format("0.0a"));
+    //setMintPercent(numeral((mintPercent)).format("0.00a"));
```

Comment:
@zashton the `mintPercent` (which was renamed `projectedMintPercent`) shouldnt it return 0 and not 10?
or is it by default 10 returning from the contract

Reply:
@jxdv yeah the contract sets it at 10% , it will not change unless the contract gets updated so thats why i named it `mintPercent` instead of `projectedMintPercent` . `projectedMint` should be 0 ,  
hmmm .. i couldn't find `projectedMintPercent` , it should be a typo if it exists.

Reply:
ah, i did the rename to `projectedMintPercent`
ok so if it will always be 10% even when no 10% mint then we will need to update the contract when possible later time, and so we can manually get this if 0 or 10 knowing it from rebase type positive vs negative for the moment, so for now its ok we will add this function-check later on simply in the sdk code

Reply:
On that note, I think we could move the `getMintPercent` to `fetchOnce`

Reply:
@jxdv yeah no problem within the contract the minting only triggers on a positive rebase, so its all good.